docs/customization/menu.rst

```diff
@@ -238,23 +245,32 @@ type is default to make the example easily customizable.
 
     namespace AppBundle\Menu;
 
-    use Sylius\Bundle\UiBundle\Menu\Event\MenuBuilderEvent;
+    use Sylius\Bundle\AdminBundle\Event\OrderShowMenuBuilderEvent;
+    use Sylius\Component\Order\Model\OrderInterface;
 
     final class AdminOrderShowMenuListener
     {
         /**
-         * @param MenuBuilderEvent $event
+         * @param OrderShowMenuBuilderEvent $event
          */
-        public function addAdminOrderShowMenuItems(MenuBuilderEvent $event)
+        public function addAdminOrderShowMenuItems(OrderShowMenuBuilderEvent $event)
         {
             $menu = $event->getMenu();
-
-            $menu->addChild('new', ['route' => 'sylius_admin_order_index'])
-                ->setAttribute('type', 'link')
-                ->setLabel('Maroon Menu Button')
-                ->setLabelAttribute('icon', 'star')
-                ->setLabelAttribute('color', 'maroon')
-            ;
+            $order = $event->getOrder();
+            $stateMachine = $event->getStateMachine();
+
+            if ($stateMachine->can(OrderTransitions::TRANSITION_FULFILL)) {
+                $menu
+                    ->addChild('fulfill', [
+                        'route' => 'sylius_admin_order_fulfill',
```

Comment:
As there is no such route in Sylius, you should give a warning about it before the example.

```
.. warning::

    There is no ``sylius_admin_order_fulfill`` route in Sylius. Create this route if you need it. 
```